scripts/csv_debug_tools/correct_column_count.py

```diff
@@ -0,0 +1,7 @@
+def get_correct_column_count(file_path):
```

Comment:
not exactly requesting a change here since these are clearly helper methods. But would be nice to generalize these for use in other scripts, or make them command-line friendly (specifying input files, etc)

Reply:
requested change impemented